kimchi/src/circuits/polynomials/turshi.rs

```diff
@@ -208,7 +208,8 @@ impl<F: PrimeField + SquareRootField> CircuitGate<F> {
         alphas.register(ArgumentType::Gate(self.typ), Instruction::<F>::CONSTRAINTS);
 
         // Get constraints for this circuit gate
-        let constraints = circuit_gate_combined_constraints(self.typ, &alphas);
+        let constraints =
+            circuit_gate_combined_constraints(self.typ, &alphas, &mut Cache::default());
```

Comment:
I guess this isn't using the same one, but it doesn't matter because we're not using turshi

Reply:
If so, then it could be source of future bugs, so maybe comment this as TODO?

Reply:
Actually, I guess this old school verify function will be replaced anyway so nvm